Week3/homework/step2-3.js

```diff
@@ -29,10 +35,19 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 // Use a 'do...while' loop
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  let result;
+  const repeatArr = [];
+  let i = 0;
+  if (i >= 0) {
```

Comment:
The condition of this `if` statement will always be `true` (i.e. `i >= 0`) because you set `i` to zero just before this `if` statement. Therefore the do-loop will always be executed once. You need to test `num`, not `i` before entering the loop.

Reply:
I made it only ' > '